.pre-commit-config.yaml

```diff
@@ -1,20 +1,20 @@
 repos:
   - repo: "https://github.com/PyCQA/flake8"
-    rev: "3.9.2"
+    rev: 5.0.4
```

Comment:
use strings to avoid yaml magic biting us at some version in future

Reply:
Done